paddle/majel/ddim/ddim.cc

```diff
@@ -1,4 +1,4 @@
-#include "paddle/majel/ddim.h"
+#include "paddle/majel/ddim/ddim.h"
```

Comment:
I think you are right for the complicated case, e.g., paddle/parameter.  For this case, let's move ddim back to `paddle/majel`.